tools/nd_freeze_install

```diff
@@ -0,0 +1,392 @@
+#!/usr/bin/python3
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+#
+# Script to install older packages based on version number.
+#
+# If running this in a stripped down environment such as a docker container,
+# install python dependencies with this command:
+#
+# apt-get update && apt-get install -y python3 python3-pip && pip3 install requests bs4
+#
+import argparse
+import glob
+import gzip
+import logging
+import os
+import re
+import requests
+import subprocess
+from bs4 import BeautifulSoup
```

Comment:
note -- current implementation requires `requests` and `bs4` non-standard libraries, which need to be installed.